src/families/polkadot/bridge/js.js

```diff
@@ -0,0 +1,102 @@
+// @flow
+import { BigNumber } from "bignumber.js";
+
+// import invariant from "invariant";
+import type { Account, Operation, SignedOperation } from "../../../types";
+import type { Transaction } from "../types";
+
+import type { AccountBridge, CurrencyBridge } from "../../../types";
+// import { isInvalidRecipient } from "../../../bridge/mockHelpers";
+import {
+  makeSync,
+  makeScanAccounts,
+  makeAccountBridgeReceive,
+} from "../../../bridge/jsHelpers";
+import { submitExtrinsic } from "../../../api/polkadot";
+import { getAccountShape } from "../synchronisation";
+
+import getTransactionStatus from "../js-getTransactionStatus";
+import estimateMaxSpendable, {
+  estimateAmount,
+} from "../js-estimateMaxSpendable";
+import signOperation from "../js-signOperation";
+
+import { preload, hydrate } from "../preload";
+
+import { calculateFees } from "../js-getFeesForTransaction";
+import { patchOperationWithHash } from "../../../operation";
+
+const receive = makeAccountBridgeReceive();
+
+const postSync = (initial: Account, parent: Account) => {
+  return parent;
+};
+
+const scanAccounts = makeScanAccounts(getAccountShape);
+
+const sync = makeSync(getAccountShape, postSync);
+
+export const createTransaction = (): Transaction => ({
+  family: "polkadot",
+  mode: "send",
+  amount: BigNumber(0),
+  recipient: "",
+  useAllAmount: false,
+  fees: null,
+  validators: [],
+  era: null,
+  rewardDestination: null,
+});
+
+const updateTransaction = (t, patch) => ({ ...t, ...patch });
+
+const sameFees = (a, b) => (!a || !b ? a === b : a.eq(b));
+
+const prepareTransaction = async (a, t) => {
+  let fees = t.fees;
+
+  fees = await calculateFees({
+    a,
+    t: {
+      ...t,
+      recipient: "1Z4QdzRrpVbggYoGK5pfbeMyzpVVDK7WxheVjWFxfv6sxjV", //Empty account to calculate fees
```

Comment:
- first of all, i'm curious why we do this here. I expect fees to not be calculated if you don't have set a recipient (consistent with other impl), it's ok tho to keep it (because in any case, you need it to probably estimate max) 
- just a good internal practice, but we prefer this to use a "public seed" so we can "prove" that we don't own this address. we usually will just use the `abandon ..` test seed to make it clear.
- also you can put it in here => https://github.com/LedgerHQ/ledgerjs/blob/master/packages/cryptoassets/src/abandonseed.js

Reply:
I suggested @henri-ly to explicitely have a non-existing account and visually understandable as such.
Good idea to put this in an abandon address.

This is because sometimes, user fills an erroneous address as recipient... this breaks fees estimation.
Also, if not filled, this can lead to shorter transactions, giving wrong fees.
I suggested to hardcode address in this case, so that it could also benefit fees estimation cache (since changeing recipient won't affect fees).